app/domain/Explorer.js

```diff
@@ -21,6 +21,7 @@ export function getDefaultExplorer(): ExplorerType {
 export function getExplorers(): Array<{| value: ExplorerType, label: string |}> {
   if (environment.isShelley()) {
     return Object.keys(ShelleyExplorers)
+      .filter(explorer => ShelleyExplorers[explorer] === ShelleyExplorers.JORMUNGANDR)
```

Comment:
@nicarq I disabled Seiza for Shelley like we did for the balance check so that it always links people to the IOHK explorer. Is that okay?

Reply:
@SebastienGllmt yeah